src/views/GRTSystemIntakeReview/index.tsx

```diff
@@ -17,30 +27,160 @@ export const GRTSystemIntakeReview = () => {
   const systemIntake = useSelector(
     (state: AppState) => state.systemIntake.systemIntake
   );
+  const isLoading = useSelector(
+    (state: AppState) => state.systemIntake.isLoading
+  );
+  const error = useSelector((state: AppState) => state.systemIntake.error);
   return (
     <div>
       <Header />
       <MainContent className="grid-container">
         <div className="system-intake__review margin-bottom-7">
           <h1 className="font-heading-xl margin-top-4">CMS System Request</h1>
-          {!systemIntake && (
+          {/* TODO: systemIntake is always truthy */}
+          {error && (
             <h2 className="font-heading-xl">
               System intake with ID: {systemId} was not found
             </h2>
           )}
-          {systemIntake && <SystemIntakeReview systemIntake={systemIntake} />}
-          <hr className="system-intake__hr" />
-          <h2 className="font-heading-xl">What to do after reviewing?</h2>
-          <p>Email the requester and:</p>
-          <ul className="usa-list">
-            <li>Ask them to fill in the business case and submit it</li>
-            <li>
-              Tell them what to expect after they submit the business case
-            </li>
-            <li>And how to get in touch if they have any questions.</li>
-          </ul>
+          {!isLoading && !error && (
+            <SystemIntakeReview systemIntake={systemIntake} />
+          )}
         </div>
       </MainContent>
+      {!isLoading && !error && (
+        <div className="bg-gray-5 padding-top-6 padding-bottom-5">
+          <MainContent className="grid-container">
```

Comment:
There can only be one `MainContent` on the page. `MainContent` is a HTML `main` tag and has a unique id. It is important to only have one for W3C standards and accessibility.